cfme/tests/ansible/test_embedded_ansible_automate.py

```diff
@@ -419,3 +421,43 @@ def test_variable_pass(request, appliance, setup_ansible_repository, import_data
         provision_request = appliance.collections.requests.instantiate(request_description)
         provision_request.wait_for_request(method="ui")
         request.addfinalizer(provision_request.remove_request)
+
+
+@pytest.mark.tier(1)
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.meta(automates=[1677575])
+@pytest.mark.parametrize(
+    "import_data",
+    [DatastoreImport("bz_1677575.zip", "bz_1677575", None)],
+    ids=["datastore"],
+)
+def test_import_export_domain_with_ansible_method(request, appliance, setup_ansible_repository,
+                                                  import_data):
+    """This test case tests support of Export/Import of Domain with Ansible Method
+
+    Bugzilla:
+        1677575
```

Comment:
I think this is redundant now, when we are using the automates marker

Reply:
I didn't hear this before. I am adding as it is part of doc string.

Reply:
Its certainly optional in the docblocks. automates/coverage/blockers meta marks are actually used by the scripting tools.  The docblock is a good place to put BZ's that are related to the case, but aren't necessarily part of automates/blockers.